apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -2,20 +2,49 @@
  * Generated by `createservice billing.BillingReceiptsService --type queries`
  */
 
+const { pick, get } = require('lodash')
+const Big = require('big.js')
+
 const { ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
-const { BillingReceipt } = require('@condo/domains/billing/utils/serverSchema')
-const { GQLCustomSchema } = require('@core/keystone/schema')
+const { Payment } = require('@condo/domains/acquiring/utils/serverSchema')
+
+const { BillingReceipt } = require('../utils/serverSchema')
 const access = require('../access/AllResidentBillingReceipts')
-const { generateQuerySortBy } = require('@condo/domains/common/utils/codegeneration/generate.gql')
-const { generateQueryWhereInput } = require('@condo/domains/common/utils/codegeneration/generate.gql')
-const { pick, get } = require('lodash')
+const { PAYMENT_DONE_STATUS } = require('@condo/domains/acquiring/constants/payment')
 const {
     BILLING_RECEIPT_RECIPIENT_FIELD_NAME,
     BILLING_RECEIPT_TO_PAY_DETAILS_FIELD_NAME,
     BILLING_RECEIPT_SERVICES_FIELD,
-} = require('@condo/domains/billing/constants/constants')
+} = require('../constants/constants')
+const { generateQuerySortBy } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+const { generateQueryWhereInput } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+
 
-const fieldsObj = {
+/**
+ * Sums all DONE payments for billingReceipt for <organization> with <accountNumber> and <period>
+ * @param context {Object}
+ * @param organizationId {string}
+ * @param accountNumber {string}
+ * @param period {string}
+ * @return {Promise<*>}
+ */
+const getPaymentsSum = async (context, organizationId, accountNumber, period) => {
+    const payments = await Payment.getAll(
+        context,
+        {
+            organization: { id: organizationId },
+            accountNumber: accountNumber,
+            period: period,
+            status: PAYMENT_DONE_STATUS,
+        }
+    )
+    return payments.reduce((total, current) => (Big(total).plus(current.amount)), 0).toFixed(8).toString()
```

Comment:
if Total is already Big-wrapped there's no need to Wrap it again...so:
```suggestion
    return payments.reduce((total, current) => (total.plus(current.amount)), Big(0)).toFixed(8).toString()
```